support/ci/build_component.ps1

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env powershell
+
+#Requires -Version 5
+
+param (
+    # The name of the component to be built. Defaults to none
+    [string]$Component
+)
+
+# Since we are only verifying we don't have build failures, make everything
+# temp!
+$env:HAB_ORIGIN="throwaway"
+# let's make a selfcontained tempdir for this job
+$job_temp_root = mkdir (Join-Path $env:TEMP ([System.IO.Path]::GetRandomFileName()))
+$env:HAB_CACHE_KEY_PATH="$job_temp_root/keys"
+
+dir
```

Comment:
did you mean to keep this `dir` command?

Reply:
hah!